glue_jupyter/bqplot/histogram/__init__.py

```diff
@@ -1,2 +1,6 @@
 from .layer_artist import *  # noqa
 from .viewer import *  # noqa
+
+
+def setup():
+    from viewer import BqplotHistogramView # noqa
```

Comment:
I don't think this is needed, since * is already imported

Reply:
Thanks. I think I got confused because I forgot that you have to manually rerun pip install to rebuild the entry points when I was developing this. And yes, this means we can simplify glue-solara to just build a list of viewers from the registry.